src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -58,6 +58,8 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
 
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Test-Connection.Ping.Ping fake host
```powershell
Expected an exception, with FullyQualifiedErrorId 'TestConnectionException,Microsoft.PowerShell.Commands.TestConnectionCommand' to be thrown, but no exception was thrown.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1: line 65
65:                 Should -Throw -ErrorId "TestConnectionException,Microsoft.PowerShell.Commands.TestConnectionCommand"
```